views/admin/css/exhibits.css

```diff
@@ -803,3 +826,32 @@ input[type=submit].configure-button {
   to {
     transform: rotate(360deg); } 
 }
+
+#cover-image-form .column select {
```

Comment:
These `#cover-image-form` styles look like they went with the leftover partial you removed.